<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>find_book_by_author</class>
 <widget class="QWidget" name="find_book_by_author">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>351</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>351</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Segoe Script</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Введите имя автора:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineAuthor">
         <property name="font">
          <font>
           <family>Segoe Script</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="findAuthor">
         <property name="font">
          <font>
           <family>Segoe Script</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Найти автора</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Segoe Script</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Выбранный автор:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ChengeAuthor">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <family>Segoe Script</family>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="show_books">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>151</width>
               <height>31</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>151</width>
               <height>31</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Segoe Script</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Показать книги</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Segoe Script</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Выберите автора:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QColumnView" name="tableView"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="back">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>23</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe Script</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="minimumSize">
          <size>
           <width>43</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>43</width>
           <height>23</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
